firmware/scripts/minify.py

```diff
@@ -19,6 +19,11 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
+
+# TODO: what this script do?
+"""
+Some documentation about this script
```

Comment:
It is self explanatory I think, it minifies the files passed as arguments. The output is the file minified (without extra new lines or code comments)

Reply:
I think i wanted more details how this is done (and projected it), sorry